community/logcheck/logcheck.cron.sample.d

```diff
@@ -0,0 +1,35 @@
+# /etc/cron.d/logcheck: crontab entries for the logcheck package
+ 
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+MAILTO=root
+ 
+@reboot         logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck -R; fi
+2 * * * *       logcheck    if [ -x /usr/sbin/logcheck ]; then nice -n10 /usr/sbin/logcheck; fi
+
+# EOF
+
+
+## busybox crond reboot script [ /etc/periodic/reboot/logcheck ]
+
+#!/bin/sh
```

Comment:
I’d just like to know how it works, when it is executed; I’m curious and can’t figure out right keywords to google it. :)

If it doesn’t work on Busybox crond, then it should be mentioned in the comment.